pkg/controllers/vdb/uninstall_reconciler.go

```diff
@@ -121,31 +123,16 @@ func (s *UninstallReconciler) uninstallPodsInSubcluster(ctx context.Context, sc
 	startPodIndex, endPodIndex int32) (ctrl.Result, error) {
 	podsToUninstall, requeueNeeded := s.findPodsSuitableForScaleDown(sc, startPodIndex, endPodIndex)
 	if len(podsToUninstall) > 0 {
-		basePod, err := findATBasePod(s.Vdb, s.PFacts)
-		if err != nil {
-			return ctrl.Result{}, err
-		}
-		ipsToUninstall := []string{}
-		for _, p := range podsToUninstall {
-			ipsToUninstall = append(ipsToUninstall, p.podIP)
-		}
-		atConfTempFile, err := s.ATWriter.RemoveHosts(ctx, basePod, ipsToUninstall)
-		if err != nil {
-			return ctrl.Result{}, err
-		}
-		defer os.Remove(atConfTempFile)
+		var result ctrl.Result
+		var err error
 
-		if err := distributeAdmintoolsConf(ctx, s.Vdb, s.VRec, s.PFacts, s.PRunner, atConfTempFile); err != nil {
-			return ctrl.Result{}, err
+		if vmeta.UseVClusterOps(s.Vdb.Annotations) {
+			result, err = s.uninstallPodsInSubclusterForVClusterOps(ctx, podsToUninstall)
+		} else {
+			result, err = s.uninstallPodsInSubclusterForAdmintools(ctx, podsToUninstall)
 		}
-
-		// Remove the installer indicator file so that we do an install if we then
-		// opt to scale out again.
-		cmd := s.genCmdRemoveInstallIndicator()
-		for _, pod := range podsToUninstall {
-			if _, _, err := s.PRunner.ExecInPod(ctx, pod.name, names.ServerContainer, cmd...); err != nil {
-				return ctrl.Result{}, fmt.Errorf("failed to call remove installer indicator file: %w", err)
-			}
+		if err != nil {
```

Comment:
We should use IsReconcileAborted
```suggestion
		if verrors.IsReconcileAborted(res, err) {
```